core/commands/root.go

```diff
@@ -36,12 +37,20 @@ Tool commands:
     version       Show ipfs version information
     commands      List all available commands
     id            Show info about ipfs peers
+    pin           Pin objects to local storage
+    name          Publish or resolve IPNS names
+    log           Change the logging level
+
+Network commands:
+
+    swarm         Manage connections to the p2p network
+    bootstrap     Add or remove bootstrap peers
+    ping          Measure the latency of a connection
```

Comment:
the `Network Commands` section should be after `Advanced` because they all require daemon